src/marshmallow/decorators.py

```diff
@@ -116,6 +120,10 @@ def post_dump(fn=None, pass_many=False, pass_original=False):
 
     If ``pass_original=True``, the original data (before serializing) will be passed as
     an additional argument to the method.
+
+    .. versionchanged:: 3.0.0
+        ``partial`` and ``many`` are always passed as keyword arguments to
+        the decorated method.
```

Comment:
Shouldn't this be added to `pre_dump` and `pre_load` as well?

Apart from this, LGTM.

Reply:
Also, AFAIU, `partial` only applies to `pre_load` / `post_load`.

Reply:
Fixed; good catch.